.github/workflows/unit-tests.yml

```diff
@@ -8,8 +8,8 @@ on:
 
 env:
   BUILD_TYPE: Debug
-  THUNDER_REF: "R2-v1.14"
-  INTERFACES_REF: "R2-v1.14"
+  THUNDER_REF: "39c82df24c7b4a4a10663cf527a3b289e7c1ceae"
+  INTERFACES_REF: "8d3bd0ddd187c6104543699041ec53e1e186d49a"
```

Comment:
Checking why do we need this version. R2-v1.14 is later than 8d3bd0ddd187c6104543699041ec53e1e186d49a. Pls check https://github.com/rdkcentral/ThunderInterfaces/commits/R2